src/main/java/dev/jbang/Util.java

```diff
@@ -733,33 +733,31 @@ public static String runCommand(String... cmd) {
 		return null;
 	}
 
-	public static boolean deleteFolder(Path folder, boolean quiet) {
-		boolean result[] = new boolean[] { true };
-		if (Files.isDirectory(folder)) {
-			Util.verboseMsg("Deleting " + folder);
-			try {
-				Files	.walk(folder)
+	public static boolean deletePath(Path path, boolean quiet) {
+		Exception err[] = new Exception[] { null };
```

Comment:
Does it need to be an array?

Reply:
Yes, unfortunately this is the trick one has to use when updating a var from within a lambda block.

Reply:
Interesting hack to know. ty =D